src/log.py

```diff
@@ -0,0 +1,55 @@
+"""
+Data Log
+
+By: Norma Chang
+"""
+import json
+import time
+
+def log(data, data_type):
```

Comment:
We should have a default value if the function doesn't receive a data type.

Reply:
I don't think we even need the data type.